### PR TITLE
Drau/link newwindow

### PR DESCRIPTION
**Pull request by daverau-optimizely**

### CODE REVIEW

File: src/components/Link/index.js [hunk 1]
```diff
@@ -43,6 +45,8 @@ Link.propTypes = {
   href: PropTypes.string,
   /** Disable link and interactivity */
   isDisabled: PropTypes.bool,
+  /** new */
```
Comment (circAssimilate) [inline src/components/Link/index.js]: Maybe we can make this "Open link in new window" instead?

File: src/components/Link/index.js [hunk 2]
```diff
@@ -30,6 +31,7 @@ const Link = ({
       data-test-section={ testSection }
       data-track-id={ testSection }
       onClick={ onClick }
+      {...(newWindow ? { target: '_blank', rel: 'noopener' } : {})}
```
Comment (circAssimilate) [inline src/components/Link/index.js]: nit: just add spaces on both sides of these brackets I'd say:

`{ ...(newWindow ? { target: '_blank', rel: 'noopener' } : {}) }`

Comment (daverau-optimizely) [reply src/components/Link/index.js]: 👍

File: src/components/Link/index.js (hunk 1 above)
Comment (daverau-optimizely) [reply src/components/Link/index.js]: 👍

**Pull request merged**